dashboard/app/models/concerns/concept_difficulties.rb

```diff
@@ -0,0 +1,20 @@
+module ConceptDifficulties
```

Comment:
My PR seems to be duplicating some of this code. Let me know whether you would prefer refactoring one of the PRs before submission or if you would prefer submitting both PRs and cleaning up the common code.